profiles/edge/default.nix

```
@@ -5,6 +5,7 @@
  */
 let
     pkgs = import (import ../../pins/18.09.nix) {};
+    mpkgs = import (import ../../pins/19.09.nix) {};
```

Oh, cool, so it's now in a numbered release. :+1:

In the other profiles, when importing pkgs from multiple releases, they use a convention of `pkgs_YYMM`. ([example](https://github.com/totten/bknix/blob/master/profiles/min/default.nix#L6-L10))

```suggestion
    pkgs_1909 = import (import ../../pins/19.09.nix) {};
```